recipes/salad/Fatoush.md

```diff
@@ -1,5 +1,17 @@
 # Fatoush Salad
 
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-70%25-orange.svg)
```

Comment:
I would agree that the reception was very positive but I think the difficulty could be lowered, 50% say? I guess the frying the bread is slightly annoying.